assets/js/components/PremiumPill/PremiumPill.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PremiumPill from '.';
+
+export default {
+  title: 'PremiumPill',
+  component: PremiumPill,
+};
+
+export function PremiumPillDefault() {
+  return <PremiumPill />;
+}
+
+export function PremiumPillRed() {
+  return <PremiumPill className="bg-red-100" />;
+}
+
+export function PremiumPillYellow() {
+  return <PremiumPill className="bg-yellow-100 text-black" />;
```

Comment:
The text colour class here has no effect... with that being said I would suggest a dark yellow text class like `text-yellow-600`. That way the text and background colours are related. See below for a reference screenshot which includes a red version too.
<img width="1045" alt="Screenshot 2023-04-06 at 15 07 10" src="https://user-images.githubusercontent.com/40714533/230388529-0b215814-5f50-4ad7-b88e-c9effbdf4d26.png">

Reply:
Yeah good idea :D